src/components/forms/OperationForm.vue

```diff
@@ -154,6 +165,36 @@ export default defineComponent({
       }
       return group
     },
+    createOptionTransfer (accounts: Account[]): GroupSelectOption {
+      const group: GroupSelectOption = {
+        label: this.$t('I18N_TRANSFER'),
+        options: []
+      }
+      for (const account of accounts) {
+        if (account.id !== this.accountId) {
+          const option: SelectOption = { value: account.id, label: account.name }
+          group.options.push(option)
+        }
+      }
+      return group
+    },
+    categoryForTransfer () {
+      if (this.account) {
+        for (const account of this.$store.state.accounts) {
+          if (this.categoryId === account.id) {
+            if (this.incoming) {
+              OperationService.addOperation(this.$store, this.categoryId, Time.getDayFromDateString(this.date), transfertCategoryId, this.signedCentsAmount * -1, this.memo + this.$t('TRANSFER_TO') + this.account.name)
```

Comment:
too many loop is a red flag. You don't really need a for loop since you only want one account. You could use the getByAccountId mentionned above. that would get rid of both the for line 183 and the if line 184

Reply:
this function does both transfer from and transfer to, you could extract both side of the if this.incoming in a function that would take the amount, source and destination and create the needed step